source/crud/tests/unified/aggregate-merge.yml

```diff
@@ -0,0 +1,188 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: aggregate-merge
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name test_aggregate_merge
+  -
+    collection:
+      id: &collection_readConcern_majority collection_readConcern_majority
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "majority" }
+  -
+    collection:
+      id: &collection_readConcern_local collection_readConcern_local
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "local" }
+  -
+    collection:
+      id: &collection_readConcern_available collection_readConcern_available
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "available" }
+runOnRequirements:
+  -
+    minServerVersion: 4.1.11
```

Comment:
I assume we don't have much control over the YAML produced by `mongodb-spec-converter`, but is it possible to do without the line break and indent for array elements (e.g. `- minServerVersion: 4.1.11` here instead)? That would be consistent with most of our hand-written files and save space.

Additionally, I'd propose ordering the top-level fields in the tests as:

 * description
 * schemaVersion
 * runOnRequirements
 * createEntites
 * initialData
 * tests

Having `runOnRequirements` up near the top helps when scanning test files.

Reply:
I can try to influence the order, but will have to see how difficult this would be.

Reply:
I've applied the order you suggested.